src/components/selectInput/SelectInput.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React, { useState } from 'react';
+import arrow from '../../assets/arrow.svg';
+import './selectInput.css';
+
+export const SelectInput = (Props: any) => {
```

Comment:
the props are defined we don't accept any as type 

interface SelectProps{ label:string, values:any[] }